src/core/a-entity.js

```diff
@@ -525,6 +525,25 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  /**
+   * A mapping of attribute to handle component attributes and singular attributes
+   *
+   * @param {string} attr dot notation or singular 'color' or 'material.color'
+   * @returns {object|string|number} Value of type of component property.
+   */
+  getComputedAttributeFor: {
```

Comment:
This has been updated, but the PR is failing to update on the branch push.  Looks like github status at the moment is saying some backend services unavailable.  Will recheck when they report servers are running normally. 

If the issue doesn't repair I will close and open a new PR.